platform/api.scripting/external/graal-sdk-1.0.0-rc6-license.txt

```diff
@@ -0,0 +1,26 @@
+Name: Graal SDK and Truffle API
+Description: Graal SDK and Truffle API
+License: UPL
+Origin: https://github.com/oracle/graal
+Version: 1.0
+Files: graal-sdk-1.0.0-rc6.jar
+OSR: not needed
+
+Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+
+The Universal Permissive License (UPL), Version 1.0
```

Comment:
According to https://search.maven.org/artifact/org.graalvm/graal-sdk/1.0.0-rc6/jar I see a different license:

>       <name>GNU General Public License, version 2, with the Classpath Exception</name>
>       <url>http://openjdk.java.net/legal/gplv2+ce.html</url>

Reply:
Fixed in version RC10: https://search.maven.org/artifact/org.graalvm.sdk/graal-sdk/1.0.0-rc10/jar